client/blocks/comments/form.scss

```diff
@@ -89,6 +84,22 @@
 	}
 }
 
+.comments__comment-list {
+	.comments__form {
+		button {
+			margin: 16px 0 -19px;
+		}
+	}
+}
+
+.conversations__comment-list {
+	.comments__form {
+		button {
+			margin: 16px 0 -9px;
+		}
+	}
+}
```

Comment:
Rather than re-define the entire `margin`, can we:
1. Use `margin-bottom: -9px`
2. Leave a comment about why this CSS is necessary